todo_app/templates/index.html

```diff
@@ -11,8 +11,21 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
+        {% for item in items %}
+          <li>{{ item.title }}</li>
+        {% endfor %}
         <!-- List the todo items here -->
       </ul>
     </div>
   </div>
+
+  <form action="/newitem" method="POST" class="form_newitem">
+    <div class="form_newitem">
+      <label for="name">Enter new todo item name: </label>
```

Comment:
The `for` of the `label` should match the `id` of the input you want to bind it to.

If done correctly, clicking on the label will focus on the input (e.g. so you can start typing in a text input)